proposals/lip-0050.md

```diff
@@ -138,22 +178,50 @@ reclaimParamsSchema = {
 
 ##### Verification
 
-For a reclaim transaction `trs` to be valid, the legacy accounts substore must contain an entry with store key equal to `getLegacyAddress(trs.senderPublicKey)` and store value `{"balance": trs.params.amount}`, serialized using `legacyAccountsSchema`.
+```python
```

Comment:
not related to this PR, but we could remove `amount` from the param (ie: use empty param)

Reply:
I agree with Shu, it doesn't make sense to pass a value for `amount` but fail verification if legacy account balance is not equal to `amount`.

Reply:
I think this is an artifact from the time that this information was needed for undoing the transaction. I agree that we could remove it now.

Reply:
With the new events, it is indeed not needed. However, there is no harm in leaving it there and I think I would rather not change the transaction params unless there is a good reason to do so.

Reply:
Sure, we could do that.

Reply:
As concluded in our tactical meeting, we will not change this command.